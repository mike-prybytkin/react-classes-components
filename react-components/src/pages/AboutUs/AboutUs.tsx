import React from 'react';

const AboutUs = () => {
  return (
    <>
      <h1>О нас!</h1>
    </>
  );
};

export default AboutUs;
