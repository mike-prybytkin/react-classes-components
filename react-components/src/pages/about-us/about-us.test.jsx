import React from 'react';
import { render, screen } from '@testing-library/react';
import AboutUs from './about-us';
import mockText from '../../mocks/text';

describe('About Us component', () => {
  it('should correctly render heading and create snapshot', () => {
    const { asFragment } = render(<AboutUs heading={mockText.headingAboutUs} />);
    const aboutUsHeading = screen.getByText(new RegExp(mockText.headingAboutUs, 'i'));

    expect(aboutUsHeading).toBeInTheDocument();
    expect(asFragment()).toMatchSnapshot();
  });
});
