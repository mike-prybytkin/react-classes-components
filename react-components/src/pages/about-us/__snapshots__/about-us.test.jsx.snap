// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`About Us component should correctly render heading and create snapshot 1`] = `
<DocumentFragment>
  <h1>
    About Us!
  </h1>
  <p
    class="sorry-message"
  />
</DocumentFragment>
`;
